include/apple2.h

```diff
@@ -232,6 +232,11 @@ struct tm* __fastcall__ gmtime_dt (const struct datetime* dt);
 time_t __fastcall__ mktime_dt (const struct datetime* dt);
 /* Converts a ProDOS date/time structure to a time_t UNIX timestamp */
 
+typedef struct DIR DIR;
```

Comment:
I'm unsure, but I believe that the usual cc65 style is to rather include dirent.h than to duplicate a typedef (?)